src/test/kotlin/no/nav/familie/ef/sak/api/gui/VedtakControllerTest.kt

```diff
@@ -330,15 +330,18 @@ internal class VedtakControllerTest : OppslagSpringRunnerTest() {
                 )
             lagTilkjentYtelse(behandlingId = behandlingId, andelerTilkjentYtelse = listOf(andel))
         }
+
         val behandlingId =
             opprettBehandling(
                 steg = StegType.SEND_TIL_BESLUTTER,
-                vedtakResultatType = ResultatType.AVSLÅ,
+                vedtakResultatType = ResultatType.INNVILGE,
```

Comment:
Kan denne testen ha to behandlinger? En innvilge og én avslå - og en sjekk på at "avslå" inneholder mulighet for inntektsoppgave?

Reply:
Litt knot med testoppsett for det, så prøver dette i en egen branch og merger